css/ai.css

```diff
@@ -228,4 +204,61 @@
     }
 }
 
-@tailwind utilities;
\ No newline at end of file
+#content a.button-next-page {
+    @apply btn btn-primary text-primary-content font-medium shadow-md shadow-accent pt-0 pb-0 pr-8 pl-8 no-underline 
+}
+
+#content h4 {
+    @apply text-xl font-medium mt-6 mb-3 text-primary dark:text-secondary
+}
+
+#content p {
+    @apply leading-6 font-normal text-inherit
+}
+
+.inner {
+    @apply prose pl-5 pr-5 md:pl-0 md:pr-0 sm:max-w-3xl text-black dark:text-white bg-base-100 my-auto mx-auto dark:bg-base-100
+}
+
+#content h2 {
+    @apply text-3xl font-medium mt-9 mb-5 text-inherit
+}
+
+/* blue bar above the page headers*/
+#content h2::before {
+    content: '';
+    display: block;
```

Comment:
I think we can use Tailwind's `block` class here.

Class | Properties
:----|:----
`block` | `display: block;`